src/index.ts

```diff
@@ -21,40 +24,9 @@ export type RuleColorCertCompliant = 'always' | 'off'; // Roku cert requirement
 
 export type BsLintConfig = Pick<BsConfig, 'project' | 'rootDir' | 'files' | 'cwd' | 'watch'> & {
     lintConfig?: string;
-    rules?: {
-        'assign-all-paths'?: RuleSeverity;
-        'unsafe-path-loop'?: RuleSeverity;
-        'unsafe-iterators'?: RuleSeverity;
-        'unreachable-code'?: RuleSeverity;
-        'case-sensitivity'?: RuleSeverity;
-        'unused-variable'?: RuleSeverity;
-        'consistent-return'?: RuleSeverity;
-        'no-stop'?: RuleSeverity;
-        // 'only-function'?: RuleSeverity,
-        // 'only-sub'?: RuleSeverity,
-        'inline-if-style'?: RuleInlineIf;
-        'block-if-style'?: RuleBlockIf;
-        'condition-style'?: RuleCondition;
-        'named-function-style'?: RuleFunction;
-        'anon-function-style'?: RuleFunction;
-        'aa-comma-style'?: RuleAAComma;
-        'type-annotations'?: RuleTypeAnnotations;
-        'no-print'?: RuleSeverity;
-        'no-todo'?: RuleSeverity;
-        // Will be transformed to RegExp type when program context is created.
-        'todo-pattern'?: string;
-        'eol-last'?: RuleEolLast;
-        'color-format'?: RuleColorFormat;
-        'color-case'?: RuleColorCase;
-        'color-alpha'?: RuleColorAlpha;
-        'color-alpha-defaults'?: RuleColorAlphaDefaults;
-        'color-cert'?: RuleColorCertCompliant;
-    };
-    globals?: string[];
-    ignores?: string[];
     fix?: boolean;
     checkUsage?: boolean;
-};
+} & JSONSchemaForBslintJson;
```

Comment:
It should also be possible to type check `BsLintRules` interface with `BsLintConfig['rules]` using a type helper like [CamelCaseProperties](https://github.com/sindresorhus/type-fest?tab=readme-ov-file#change-case) for the kebab to camelcase key conversion. 

Though I noticed the below 2 keys don't exactly match the conversion which would cause a small breaking change:

*BsLintConfig['rules]*
```typescript
"assign-all-paths"?: "off" | "info" | "warn" | "error";
"color-cert"?: "always" | "off";
```

*BsLintRules*
```typescript
assignAllPath: BsLintSeverity;
colorCertCompliant: RuleColorCertCompliant;
```

I left this out for now, though happy to add it if we feel the key normalisation is worth the change.